### PR TITLE
Add Crusher Glaive to the Salvage Vendor

### PR DESCRIPTION
**Pull request by AeraAuling**
## About the PR
Added two Crusher Glaives to the salvage vendor

## Why / Balance
Added because Crusher Glaives are nice, basically. They are otherwise fairly identical to regular crushers. Readding them to the vendor should also allow for some visual variety and a bit of added flavor with salvage's choice of weaponry.

## Technical details
This touches the file at /Resources/Prototypes/Catalog/VendingMachines/Inventories/salvage.yml otherwise not much else is changed.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
image

## Breaking changes

N/A
**Changelog**

-Adds Crusher Glaives to the salvage vendor, finally bringing proper polearms to the delta sector.

### CODE REVIEW

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/salvage.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@
     HandheldGPSBasic: 2
     RadioHandheld: 2
     WeaponCrusher: 2
+    WeaponCrusherGlaive: 2
```
Comment (Colin-Tel) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/salvage.yml]:
```suggestion
    WeaponCrusherGlaive: 2 # DeltaV
```

**Review by Colin-Tel — CHANGES_REQUESTED** [review]

**Comment by Colin-Tel** [comment]
Why isn't this already in? Is it because of balance?

**Comment by Adrian16199** [comment]
> Why isn't this already in? Is it because of balance?

I believe the reason is the fact that it is basicly the crusher but diferent look.
No diferent stats.

**Comment by IamVelcroboy** [comment]
Are these items supposed to be research locked?

**Comment by IamVelcroboy** [comment]
Or...should they be?

**Comment by FluffiestFloof** [comment]
> > Why isn't this already in? Is it because of balance?
> 
> I believe the reason is the fact that it is basicly the crusher but diferent look. No diferent stats.

They heal more damage when you hit a marked enemy
Crusher:
https://github.com/DeltaV-Station/Delta-v/blob/2fa1a61913e4e819edde09973e00a6d2936fcf88/Resources/Prototypes/Entities/Objects/Weapons/Melee/mining.yml#L30-L33
Glaive:
https://github.com/DeltaV-Station/Delta-v/blob/2fa1a61913e4e819edde09973e00a6d2936fcf88/Resources/Prototypes/Entities/Objects/Weapons/Melee/mining.yml#L105-L108

They have a faster melee attack speed
Crusher:
https://github.com/DeltaV-Station/Delta-v/blob/2fa1a61913e4e819edde09973e00a6d2936fcf88/Resources/Prototypes/Entities/Objects/Weapons/Melee/mining.yml#L46-L51
Glaive:
https://github.com/DeltaV-Station/Delta-v/blob/2fa1a61913e4e819edde09973e00a6d2936fcf88/Resources/Prototypes/Entities/Objects/Weapons/Melee/mining.yml#L111-L113

**Comment by Colin-Tel** [comment]
> Or...should they be?

I'd be more inclined to say that these should be research locked rather than have these available at the same time the objectively worse equipment is as well.

**Comment by AeraAuling** [comment]
> > Or...should they be?
> 
> I'd be more inclined to say that these should be research locked rather than have these available at the same time the objectively worse equipment is as well.

I suppose they could be researched too yeah. I, did not know about the more health but I will say the slightly faster attack speed doesn't play out that much in practice. They attack at about the same rate when using the shot and strike.

**Comment by DebugOk** [comment]
> Why isn't this already in? Is it because of balance?

I don't think these should be in until Wizden adds them, especially not as something available roundstart

**Comment by deltanedas** [comment]
since its better in every way i think it should be in salvage tech, and probably upstreamed

**Pull request closed without merging**